packages/injectable/src/dependency-injection-container/createContainer.registration.test.js

```diff
@@ -15,6 +16,24 @@ describe('createContainer.registration', () => {
     expect(actual).toBe('some-injected-instance');
   });
 
+  it('throws an error with the key if the value behind has no default export', () => {
```

Comment:
Test name does not comply to **gherkin** (ie. given, when, then). Should be: 'given injectable file with no default export, when auto-registering, throws with name of faulty file'. Note: as convention, word "then" is left out intentionally.